secure/key-manager/src/tests.rs

```diff
@@ -350,8 +350,15 @@ fn setup_node_using_test_mocks() -> Node<MockLibraInterface> {
 // Creates and returns a libra database and database reader/writer pair bootstrapped with genesis.
 fn setup_libra_db(config: &NodeConfig) -> (Arc<LibraDB>, DbReaderWriter) {
     let (storage, db_rw) = DbReaderWriter::wrap(LibraDB::new_for_test(&config.storage.dir()));
-    db_bootstrapper::bootstrap_db_if_empty::<LibraVM>(&db_rw, get_genesis_txn(config).unwrap())
-        .expect("Failed to execute genesis");
+    let waypoint =
+        db_bootstrapper::generate_waypoint::<LibraVM>(&db_rw, get_genesis_txn(config).unwrap())
+            .expect("Failed to execute genesis");
+    db_bootstrapper::bootstrap_genesis::<LibraVM>(
+        &db_rw,
+        get_genesis_txn(config).unwrap(),
+        waypoint,
+    )
+    .unwrap();
```

Comment:
looks like we need a helper to use in tests

Reply:
I'll try to make a test only function, the cfg(feature=testing) part makes me lazy..

Reply:
I need a emoji here. oh this one looks proper:

😝😝😝😝😝😝😝😝😝😝